segments/__init__.py

```diff
@@ -18,6 +20,15 @@ def __init__(self, *args, **kwargs):
             # Other segments are active if the config files states so.
             self.active = config.SEGMENTS.get(class_name, False)
 
+        if( config.SPACES ):
+            if class_name not in ['newline', 'root', 'divider', 'padding', 'exitcode']:
```

Comment:
It's clear to me why NOT adding right and left spaces for newline, divider, padding. But why root and exitcode?